Travis: Node `4.2` install fails

See https://travis-ci.org/FormidableLabs/formidable-react-component-boilerplate/jobs/87699535

* `/home/travis/build/FormidableLabs/formidable-react-component-boilerplate/node_modules/builder-react-component/node_modules/webpack-dev-server/node_modules/socket.io/node_modules/engine.io/node_modules/ws/node_modules/utf-8-validate/build`
* `/home/travis/build/FormidableLabs/formidable-react-component-boilerplate/node_modules/builder-react-component/node_modules/webpack-dev-server/node_modules/socket.io/node_modules/engine.io/node_modules/ws/node_modules/bufferutil`

Looks like some problem with:

```
../node_modules/nan/nan_converters_43_inl.h:24:1: error: ‘return_t’ in ‘struct Nan::imp::ToFactory<v8::Int32>’ does not name a type
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -3,8 +3,8 @@ language: node_js
 node_js:
   - "0.10"
   - "0.12"
-  # https://github.com/FormidableLabs/formidable-react-component-boilerplate/issues/53
-  #- "4.2"
+  - "4.2"
+  - "5.0"
 
 # Use container-based Travis infrastructure.
 sudo: false
@@ -13,16 +13,23 @@ branches:
   only:
     - master
 
+env:
+  matrix:
+    - NPM_3=true
+    - NPM_3=false
+
 before_install:
   # GUI for real browsers.
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
 
+  # Potentially update to npm 3
+  - 'if [ "$NPM_3" = true ]; then npm install -g npm@3; else true; fi'
+
 script:
-  - node_modules/.bin/builder install
+  - npm --version
   - node_modules/.bin/builder run check-ci
 
   # Prune deps to just production and ensure we can still build
   - npm prune --production
-  - npm install --production
   - node_modules/.bin/builder run build

--- a/DEVELOPMENT.md
+++ b/DEVELOPMENT.md
@@ -28,13 +28,6 @@ Note that `dist/` files are only updated and committed on **tagged releases**.
 
 ## Development
 
-Most development / quality tasks require the `devDependencies` from the
-`builder` archetype. Install all dev dependencies with:
-
-```sh
-$ builder install
-```
-
 All development tasks consist of watching the demo bundle, the test bundle
 and launching a browser pointed to the demo page.
 

--- a/package.json
+++ b/package.json
@@ -19,10 +19,11 @@
     "test": "builder run npm:test"
   },
   "dependencies": {
-    "builder": "^2.0.1",
-    "builder-react-component": "0.0.4"
+    "builder": "2.1.0",
+    "builder-react-component": "0.1.0"
   },
   "devDependencies": {
+    "builder-react-component-dev": "0.1.0",
     "chai": "^3.2.0",
     "mocha": "^2.3.3",
     "react": "^0.14.0",

